x-pack/plugins/apm/public/components/alerting/ui_components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -0,0 +1,464 @@
+                  gutterSize="s"
+                  responsive={false}
+                >
+                  <EuiFlexItem grow={false}>
+                    <EuiTitle size="xs">
+                      <h2>
+                        {i18n.translate(
+                          'xpack.apm.serviceOverview.throughtputChartTitle',
+                          { defaultMessage: 'Throughput' }
+                        )}
+                      </h2>
+                    </EuiTitle>
+                  </EuiFlexItem>
+
+                  <EuiFlexItem grow={false}>
+                    <EuiIconTip
+                      content={i18n.translate(
+                        'xpack.apm.serviceOverview.tpmHelp',
+                        {
+                          defaultMessage:
+                            'Throughput is measured in transactions per minute (tpm).',
+                        }
+                      )}
+                      position="right"
+                    />
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+
+                <TimeseriesChart
+                  id="throughput"
+                  height={200}
+                  comparisonEnabled={comparisonEnabled}
+                  offset={offset}
+                  fetchStatus={statusThroughput}
+                  customTheme={comparisonChartTheme}
+                  timeseries={timeseriesThroughput}
+                  yLabelFormat={getResponseTimeTickFormatter(latencyFormatter)}
+                  anomalyTimeseries={
+                    preferredAnomalyTimeseriesLatency
+                      ? {
+                          ...preferredAnomalyTimeseriesLatency,
+                          color: anomalyTimeseriesColor,
+                        }
+                      : undefined
+                  }
+                  timeZone={timeZone}
+                />
+              </EuiPanel>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiPanel hasBorder={true}>
+                <EuiFlexGroup
+                  alignItems="center"
+                  gutterSize="s"
+                  responsive={false}
+                >
+                  <EuiFlexItem grow={false}>
+                    <EuiTitle size="xs">
+                      <h2>
+                        {i18n.translate('xpack.apm.errorRate', {
+                          defaultMessage: 'Failed transaction rate',
+                        })}
+                      </h2>
+                    </EuiTitle>
+                  </EuiFlexItem>
+
+                  <EuiFlexItem grow={false}>
+                    <EuiIconTip
+                      content={i18n.translate('xpack.apm.errorRate.tip', {
+                        defaultMessage:
+                          "The percentage of failed transactions for the selected service. HTTP server transactions with a 4xx status code (client error) aren't considered failures because the caller, not the server, caused the failure.",
+                      })}
```

Comment:
should this be a shared label? I assume we use it in another place too? This shouldn't pass CI either, because the label id is duplicated.

Reply:
I exported it from _original_ failed_transaction_rate_chart. I didn't find a shared file where APM puts the shared i18n